src/screens/Home/Home.tsx

```diff
@@ -38,6 +39,45 @@ interface HomeProps {
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
 
+  const calculateCustomerData = () => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let numCustomersToCharge = 0;
+    let numOutstandingPayments = 0;
+    const customers = currentSite.customers;
+    for (let i = 0; i < customers.length; i++) {
+      const currCustomer = customers[i]
+      //depends if the meter readings list is sorted with earliest => latest
+      let latestMeterReading = currCustomer.meterReadings[currCustomer.meterReadings.length - 1];
+      if (latestMeterReading != null) {
+        let lessMonth = lessThanCurrentPeriod(latestMeterReading.date);
+        if (lessMonth) {
```

Comment:
https://github.com/calblueprint/meepanyar/pull/36#discussion_r586927473

You can put the function call inside the `if` statement since the variable is only used once if thinking of a good name is difficult here